installer/bundle/bundle_skel_Linux.sh

```diff
@@ -702,29 +581,25 @@ case "$installMode" in
 
     I)
         echo "Installing cross-platform agent ..."
-        if [ "$PLATFORM" = "SunOS" ]; then
-            check_if_pkg_is_installed scx
-            if [ $? -eq 0 ]; then
-                echo "ERROR: SCX package is already installed"
-                cleanup_and_exit 2
-            fi
-        fi
 
         if [ $PROVIDER_ONLY -eq 0 ]; then
             check_if_pkg_is_installed omi
             if [ $? -eq 0 ]; then
-                pkg_upd $OMI_PKG omi
-                # It is acceptable that this fails due to the new omi being 
-                # the same version (or less) than the one currently installed.
-                OMI_EXIT_STATUS=0
+                pkg_upd_list $OMI_PKG omi
             else
-                pkg_add $OMI_PKG omi
-                OMI_EXIT_STATUS=$?
+                pkg_add_list $OMI_PKG omi
             fi
         fi
 
-        pkg_add $OM_PKG scx
-        SCX_EXIT_STATUS=$?
+        pkg_add_list $OM_PKG scx
+
+        pkg_upd ${UPD_PKG_QUEUE}
+        # It is acceptable that this fails due to the new omi being 
+        # the same version (or less) than the one currently installed.
+        OMI_EXIT_STATUS=0
```

Comment:
> setting OMI_EXIT_STATUS to zero is just wrong (as it'll not catch SCX install issues).

SCX install issues are not relevant here.. right?  the ```UPD_PKG_QUEUE``` would not contain scx at this point.  If the script is at this line, it means the user is running the script with ```--install``` option. So, UPD_PKG_QUEUE will either be empty or will only contain omi. 

> My suggestion is just to test if it's installed (at 587) and, if it is, do the same version check that the regular upgrade path does. Then, right after 588, call pkg_upd to actually do the update (since all other code below uses pkg_add). 

OK.